### PR TITLE
monospace

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: static/css/global.css [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+    font-family: "Source Code Pro";
+    src: url("../fonts/SourceCodePro-VariableFont_wght.ttf");
+}
+
 * {
   margin: 0;
   padding: 0;
-  box-sizing: border-box
+  box-sizing: border-box;
+  font-family: "Source Code Pro", monospace;
```
Comment (mwawrzen) [inline static/css/global.css]: useless semicolon

File: static/css/global.css [hunk 2]
```diff
@@ -1,9 +1,15 @@
 @import 'variables.css';
 
+@font-face {
+    font-family: "Source Code Pro";
+    src: url("../fonts/SourceCodePro-VariableFont_wght.ttf");
```
Comment (mwawrzen) [inline static/css/global.css]: useless semicolon

**Review by mwawrzen — APPROVED** [review]
LGTM

**Review by jaanonim — APPROVED** [review]
LGTM

**Pull request merged**